src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
+        if (getBranch().getForkingMarket() != _market) {
+            return 0;
+        }
+        ReputationToken _winningDestination = getReputationToken().getTopMigrationDestination();
+        if (address(_winningDestination) == address(0)) {
+            return 0;
+        }
+        if (_winningDestination.totalSupply() < 11 * 10**6 * 10**18 / 2 && block.timestamp < getBranch().getForkEndTime()) {
```

Comment:
We should probably move this magic number into a variable, just so we can have it named.  I think constant variables are inlined in Solidity, so wouldn't increase contract size over this?